vs2017 building issue

Any one build successfully CZMQ using vs2017

Wow! @myd7349 thanks, I compiled & ran my app on Windows 10 using Visual Studio 2019 with czmq via vcpkg.  So easy, fast and simple..... Except for one issue:  

`Error CMake Error at vcpkg/installed/x86-windows/share/czmq/czmqConfig.cmake:31 (if):
  if given arguments:
    "AND" "FALSE"
  Unknown arguments specified vcpkg/installed/x86-windows/share/czmq/czmqConfig.cmake	31
`

Not sure what the logic at `czmqConfig.cmake` line 31 intends:

`find_dependency(ZeroMQ)`
`if ( AND FALSE)`
`    find_dependency(OpenSSL)`
`    find_dependency(ZLIB)`
`endif ()`


But by commenting out the `if` & `endif` lines I bypassed the error and got it to function.

--- a/ports/czmq/CONTROL
+++ b/ports/czmq/CONTROL
@@ -1,5 +1,5 @@
 Source: czmq
-Version: 2019-06-10
+Version: 2019-06-10-1
 Build-Depends: zeromq
 Description: High-level C binding for ZeroMQ
 Homepage: https://github.com/zeromq/czmq

--- a/ports/czmq/fix-dependencies.patch
+++ b/ports/czmq/fix-dependencies.patch
@@ -9,7 +9,7 @@ index 9c15f36a..e1475cd6 100644
 +
 +find_dependency(ZeroMQ)
 +
-+if (@CZMQ_WITH_LIBCURL@ AND @LIBCURL_FOUND@)
++if ("@CZMQ_WITH_LIBCURL@" AND "@LIBCURL_FOUND@")
 +    find_dependency(OpenSSL)
 +    find_dependency(ZLIB)
 +endif ()

